src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	private static final int COUNT_WHEN_FULL = 11;
```

Comment:
This constant is related to MAX_NUMBERS_COUNT. Changing only one leads to an error (inconsistency).